xikolo-ios/ViewControllers/Courses/CourseListViewController.swift

```diff
@@ -106,6 +122,15 @@ class CourseListViewController : AbstractCourseListViewController {
         }
     }
 
+    @IBAction func unwindLogin(_ segue: UIStoryboardSegue) {
```

Comment:
We could, but isn't this the textbook case for unwind segues (Especially when thinking about the sso use case)? I never liked the `presentingViewController?.dismissMe()` method